lib/nerves_hub_link/socket.ex

```diff
@@ -356,7 +356,7 @@ defmodule NervesHubLink.Socket do
   def handle_message(@console_topic, "file-data", params, socket) do
     path = Path.join(socket.assigns.data_path, params["filename"])
 
-    {:ok, _res} =
+    _res =
```

Comment:
is it worth just matching on `:ok`? looking at the docs for `File.open!`, it will return the result of the function.

Reply:
I think it should be returned to the user and I also think the bang needs to be removed. Right now the behavior is to restart the shell on every successful upload and that's what I wanted fixed.

Reply:
Actually... maybe there's more to it. I just noticed that only 1K of the file gets uploaded.

Reply:
If we switch to `_res =` then this uploads the whole 74k file I drop onto the console window. It would be 1k because that's the chunk size we do and it crashes on the first message

Reply:
Cool. That makes sense about the 1K chunk size. 

Ok, shall we go with the quick fix and then add an issue for giving better feedback through the UI for errors?